source/crud/crud.rst

```diff
@@ -766,45 +777,73 @@ Bulk Write Models
     /**
      * The filter to limit the deleted documents.
      *
-     * @see http://docs.mongodb.org/manual/reference/command/delete/
+     * @see https://docs.mongodb.org/manual/reference/command/delete/
      */
     filter: Document;
 
+    /**
+     * Specifies a collation.
+     *
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     * For servers < 3.4, the driver MUST raise an error if the caller explicitly provides a value.
```

Comment:
I didn't think so, but probably. These ones are kinda weird, but better to have it there than not.